src/System.Data.SqlClient/tests/Tools/TDS/TDS.EndPoint/SSPI/SecurityWrapper.cs

```diff
@@ -24,7 +24,7 @@ internal class SecurityWrapper
         /// <param name="pvGetKeyArgument">This parameter is not used and should be set to NULL</param>
         /// <param name="phCredential">A pointer to a CredHandle structure to receive the credential handle</param>
         /// <param name="ptsExpiry">A pointer to a TimeStamp structure that receives the time at which the returned credentials expire</param>
-        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto)]
+        [DllImport(Interop.Libraries.SspiCli, CharSet = CharSet.Auto, EntryPoint="AcquireCredentialsHandleW")]
```

Comment:
> And you probably also want to set the exact name property to true.

I thought the exact name property was only for when the actual managed function name has the same name, but in this case the name doesn't have the W, I'm just adding an entrypoint to the attribute.